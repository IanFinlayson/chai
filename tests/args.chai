def makeWindow(title = "", width = 1024, height = 768, full = False):
    print("title is '", title, "'", sep="")
    print("width is", width)
    print("height is", height)
    print("full is", full, end="\n\n")


def main():
    makeWindow("Game", 800, 600, False)
    makeWindow()
    makeWindow("My Program")
    makeWindow(title = "Tetris", full = True)

