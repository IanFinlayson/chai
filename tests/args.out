title is 'Game'
width is 800
height is 600
full is False

title is ''
width is 1024
height is 768
full is False

title is 'My Program'
width is 1024
height is 768
full is False

title is 'Tetris'
width is 1024
height is 768
full is True

